package.json

```diff
@@ -13,7 +13,8 @@
     "prepublish": "npm run build && npm run typings",
     "lint": "prettier --write src/* && eslint src --ext .ts",
     "test:unit": "jest --testPathPattern=__tests__/units --passWithNoTests",
-    "test:web": "jest --testPathPattern=__tests__/web --preset=jest-puppeteer"
+    "test:web": "jest --testPathPattern=__tests__/web --preset=jest-puppeteer",
```

Comment:
can we jest use one script with `--runInBand` ?

Reply:
I think we can run it parallelly, the previous error is caused by document is not ready and input wasn't focus.